test/TritonGPU/amd/amd-stream-lds-layout-selection.mlir

```diff
@@ -0,0 +1,142 @@
+// RUN: triton-opt %s -split-input-file -tritonamdgpu-stream-pipeline="num_stages=2" -canonicalize | FileCheck %s
+
+// Pick a common shared memory layout with vec = max kWidth of all users.
+// CHECK{LITERAL}: #shared = #ttg.swizzled_shared<{vec = 8, perPhase = 2, maxPhase = 8, order = [0, 1]}>
+// CHECK-NOT: #shared1
```

Comment:
You may want to `CHECK-NOT:` `#ttg.swizzled_shared` given `#shared1` naming is not guaranteed to be stable.